app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      render new_user_path
+    end
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to new_user_path
+    else
+      render 'new'
```

Comment:
- [ ] Prefer symbols instead of strings, it's more ruby-like.
- [ ] Here you should render the edit action, not the new action.